src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs

```diff
@@ -1,167 +1,229 @@
-        public List<SectorViewModel> Sectors { get; set; }
-        public List<string> SelectedSectors { get; set; }
-        public List<int> SelectedLevels { get; set; }
-        public OrderBy OrderBy
+        get => _orderBy;
+        set
         {
-            get => _orderBy;
-            set
+            if (value == OrderBy.None && !string.IsNullOrEmpty(Keyword))
             {
-                if (value == OrderBy.None && !string.IsNullOrEmpty(Keyword))
-                {
-                    _orderBy = OrderBy.Relevance;
-                }
-                else if (value != OrderBy.None && string.IsNullOrEmpty(Keyword))
-                {
-                    _orderBy = OrderBy.None;
-                }
-                else
-                {
-                    _orderBy = value;
-                }
+                _orderBy = OrderBy.Relevance;
+            }
+            else if (value != OrderBy.None && string.IsNullOrEmpty(Keyword))
+            {
+                _orderBy = OrderBy.None;
+            }
+            else
+            {
+                _orderBy = value;
             }
         }
+    }
 
-        public bool ShowFilterOptions => ClearSectorLinks.Any() || ClearLevelLinks.Any() || !string.IsNullOrEmpty(Keyword);
+    public bool ShowFilterOptions => SelectedFilterSections.Count > 0;
 
-        public string TotalMessage => GetTotalMessage();
-        public Dictionary<string, string> ClearSectorLinks => BuildClearSectorFilterLinks();
-        public string ClearKeywordLink => BuildClearKeywordFilterLink();
-        public Dictionary<string, string> ClearLevelLinks => BuildClearLevelFilterLinks();
-        public string OrderByName => BuildOrderByLink(OrderBy.Name);
-        public string OrderByRelevance => BuildOrderByLink(OrderBy.Relevance);
+    public string WorkLocationDisplayMessage => GetWorkLocationDistanceDisplayMessage();
 
+    public string TotalMessage => GetTotalMessage();
 
-        private string GetTotalMessage()
-        {
-            var totalToUse = string.IsNullOrEmpty(Keyword)
-                             && (SelectedSectors == null || !SelectedSectors.Any())
-                             && (SelectedLevels == null || !SelectedLevels.Any())
-                                    ? Total
-                                    : TotalFiltered;
+    private string GetTotalMessage()
+    {
+        var totalToUse = string.IsNullOrEmpty(Keyword)
+                         && (SelectedRoutes == null || SelectedRoutes.Count < 1)
+                         && (SelectedLevels == null || SelectedLevels.Count < 1)
+                                ? Total
+                                : TotalFiltered;
 
-            return $"{totalToUse} result" + (totalToUse != 1 ? "s" : "");
+        return $"{totalToUse} result" + (totalToUse != 1 ? "s" : "");
+    }
+
+    private string GetWorkLocationDistanceDisplayMessage()
+    {
+        if(!this.Distance.HasValue || this.Distance < 1 || this.Distance > 100)
```

Comment:
It is possible that there is no location but the distance is given, in this case we need to ignore the distance. I do not see this being checked here.